### PR TITLE
Adds "added by" and "added at" columns to the table, changes to API header

### PR DESCRIPTION
**Pull request by mishushakov**
- Adds "added by" and "added at" columns to the table
- Changes to API /teams/:id/users headers

Blocked by https://github.com/e2b-dev/billing/pull/25

### CODE REVIEW

File: apps/web/src/components/Dashboard/Team.tsx [hunk 1]
```diff
@@ -278,13 +284,14 @@ export const TeamContent = ({
                   key={user.id}
                 >
                   <TableCell>{user.email}</TableCell>
-                  <TableCell>
+                  <TableCell>{user.addedBy?.email}</TableCell>
+                  <TableCell>{user.addedAt}</TableCell>
```
Comment (0div) [inline apps/web/src/components/Dashboard/Team.tsx]: is this `toLocalDateTime()`ed?

File: apps/web/src/components/Dashboard/Team.tsx [hunk 2]
```diff
@@ -62,7 +66,7 @@ export const TeamContent = ({
         `${process.env.NEXT_PUBLIC_BILLING_API_URL}/teams/${team.id}/users`,
         {
           headers: {
-            'X-Team-API-Key': team.apiKeys[0],
+            'X-User-Access-Token': user.accessToken,
```
Comment (0div) [inline apps/web/src/components/Dashboard/Team.tsx:69]: i assume the server side is aware of this header change?

Comment (jakubno) [reply apps/web/src/components/Dashboard/Team.tsx:69]: I would send both for now, add an issue to remove once the new backend is deployed

Comment (mishushakov) [reply apps/web/src/components/Dashboard/Team.tsx:69]: after https://github.com/e2b-dev/billing/pull/25 is merged, yep!

Comment (mishushakov) [reply apps/web/src/components/Dashboard/Team.tsx:69]: sending both headers now

**Comment by jakubno** [comment]
Can we split it to two separate PRs? First would be only to send both API key + Access token, then merge backend and right after than we can merge this. I think it's the only possible solution how to not break anything
image

**Review by jakubno — CHANGES_REQUESTED** [review]
Wrong PR

**Comment by mishushakov** [comment]
Your screenshot shows templates view, this is for team users view

File: apps/web/src/components/Dashboard/Team.tsx [hunk 3]
```diff
@@ -278,13 +285,16 @@ export const TeamContent = ({
                   key={user.id}
                 >
                   <TableCell>{user.email}</TableCell>
```
Comment (jakubno) [inline apps/web/src/components/Dashboard/Team.tsx:287]: Smal UI thing 
image

Comment (mishushakov) [reply apps/web/src/components/Dashboard/Team.tsx:287]: ouch

**Review by jakubno — APPROVED** [review]
Looks good!

**Comment by jakubno** [comment]
But wait for API to be merged, I will do it this evening

**Pull request merged**